### ISSUE TITLE
Fix Color Contrast on Homepage Links

### ISSUE THREAD

**Issue by vkoves** [vkoves/vkoves.github.io#6]
An issue from HTML Codesniffer indicates that the homepage red links are not color contrast compliant:

![screenshot from 2018-12-09 12-59-54](https://user-images.githubusercontent.com/3187531/49701493-bbe68f80-fbb2-11e8-8e50-148d6e56a42d.png)

This should be fixed by adjusting the link color in this context.
